python/podio/frame.py

```diff
@@ -48,16 +48,38 @@ def _determine_cpp_type(idx_and_type):
 SUPPORTED_PARAMETER_TYPES = _determine_supported_parameter_types()
 
 
-def _get_cpp_vector_types(type_str):
-  """Get the possible std::vector<cpp_type> from the passed py_type string."""
-  # Gather a list of all types that match the type_str (c++ or python)
+def _get_cpp_types(type_str):
+  """Get all possible c++ types from the passed py_type string."""
   types = list(filter(lambda t: type_str in t, SUPPORTED_PARAMETER_TYPES))
```

Comment:
```suggestion
  types = [t for t in SUPPORTED_PARAMETER_TYPES if type_str in t]
```
simpler, more pythonic